vncviewer/Viewport.cxx

```diff
@@ -598,13 +600,40 @@ int Viewport::handle(int event)
   case FL_DRAG:
   case FL_MOVE:
   case FL_MOUSEWHEEL:
-    buttonMask = 0;
+
+    // FIXME: FLTK is not consistent in how it sends FL_PUSH/FL_RELEASE
+    // events:
+    //   * https://github.com/fltk/fltk/issues/1076
+    //   * https://github.com/fltk/fltk/issues/1077.
+    // For LMB/MMB/RMB, we can get the button states consistently.
+    // We can't get the state for back/forward, and have to keep track
+    // of them ourselves. Unfortunately as FLTK has these
```

Comment:
This is an important note even without FLTK bugs. Perhaps it should be separated out to be made more clear?